geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DescribeConfigCommand.java

```diff
@@ -62,62 +60,55 @@ public ResultModel describeConfig(
+    }
 
-      if (obj != null && obj instanceof MemberConfigurationInfo) {
-        MemberConfigurationInfo memberConfigInfo = (MemberConfigurationInfo) obj;
+    CliFunctionResult functionResult =
+        executeFunctionAndGetFunctionResult(getMemberConfigFunction, hideDefaults, targetMember);
+    Object obj = functionResult.getResultObject();
 
-        result
-            .setHeader(CliStrings.format(CliStrings.DESCRIBE_CONFIG__HEADER__TEXT, memberNameOrId));
+    if (obj != null && obj instanceof MemberConfigurationInfo) {
+      MemberConfigurationInfo memberConfigInfo = (MemberConfigurationInfo) obj;
 
-        List<String> jvmArgsList = memberConfigInfo.getJvmInputArguments();
-        TabularResultModel jvmInputArgs = result.addTable(JVM_ARGS_SECTION);
+      result
+          .setHeader(CliStrings.format(CliStrings.DESCRIBE_CONFIG__HEADER__TEXT, memberNameOrId));
 
-        for (String jvmArg : jvmArgsList) {
-          // This redaction should be redundant, since jvmArgs should have already been redacted in
-          // MemberConfigurationInfo. Still, better redundant than missing.
-          jvmInputArgs.accumulate("JVM command line arguments", ArgumentRedactor.redact(jvmArg));
-        }
+      List<String> jvmArgsList = memberConfigInfo.getJvmInputArguments();
+      TabularResultModel jvmInputArgs = result.addTable(JVM_ARGS_SECTION);
 
-        addSection(API_PROPERTIES_SECTION, result, memberConfigInfo.getGfePropsSetUsingApi(),
-            "GemFire properties defined using the API");
-        addSection(RUNTIME_PROPERTIES_SECTION, result, memberConfigInfo.getGfePropsRuntime(),
-            "GemFire properties defined at the runtime");
-        addSection(FILE_PROPERTIES_SECTION, result, memberConfigInfo.getGfePropsSetFromFile(),
-            "GemFire properties defined with the property file");
-        addSection(DEFAULT_PROPERTIES_SECTION, result,
-            memberConfigInfo.getGfePropsSetWithDefaults(),
-            "GemFire properties using default values");
-        addSection(CACHE_ATTRIBUTES_SECTION, result, memberConfigInfo.getCacheAttributes(),
-            "Cache attributes");
-
-        List<Map<String, String>> cacheServerAttributesList =
-            memberConfigInfo.getCacheServerAttributes();
-
-        if (cacheServerAttributesList != null && !cacheServerAttributesList.isEmpty()) {
-          for (Map<String, String> cacheServerAttributes : cacheServerAttributesList) {
-            addSection(CACHESERVER_ATTRIBUTES_SECTION, result, cacheServerAttributes,
-                "Cache-server attributes");
-          }
-        }
+      for (String jvmArg : jvmArgsList) {
+        // This redaction should be redundant, since jvmArgs should have already been redacted in
+        // MemberConfigurationInfo. Still, better redundant than missing.
+        jvmInputArgs.accumulate("JVM command line arguments", ArgumentRedactor.redact(jvmArg));
       }
 
-    } catch (FunctionInvocationTargetException e) {
```

Comment:
Why were we able to get rid of this exception handling and the catch-all below?  Does the caller do the handling now?

Reply:
yes, the caller(CommandExecutor) handles them all